db/seeds.rb

```diff
@@ -46,13 +42,11 @@
 
 10.times do
   date = Faker::Date.in_date_period
-  cat = cats.sample
-  # Avoid having the same user *owning* and *renting* a cat.
-  possible_user_ids = user_ids.select{|id| id != cat.user_id}
+  cat = Cat.all.sample
 
   booking = Booking.create!(
     cat_id: cat.id,
-    user_id: user_id = possible_user_ids.sample,
+    user_id: (User.all.select{|user| user.id != cat.user.id }).sample.id,
```

Comment:
maybe same here as for the cat, since we're at it? :-)

Booking.new and inside put just the dates
booking.cat = Cat.all.sample
booking.user = User.all.select{|user| user != booking.cat.user }).sample
booking.save!